mllib/src/main/scala/org/apache/spark/ml/param/sharedParams.scala

```diff
@@ -34,6 +34,43 @@ private[ml] trait HasRegParam extends Params {
   def getRegParam: Double = get(regParam)
 }
 
+private[ml] trait HasElasticNetParam extends HasRegParam {
```

Comment:
Do we want to call this "alphaParam?"  That name assumes people have read elastic net papers.  What about something like:
* "elasticNetParam" (very explicit) (I assume this was your original name.)
* "regMixing" (also explicit)
* "regAlpha" (at least will be grouped next to "regParam")